### PR TITLE
Shift features to webprofile-9.0

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#libby

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature [hunk 1]
```diff
@@ -7,15 +7,8 @@ IBM-ShortName: jakartaee-9.0
...
- com.ibm.websphere.appserver.jndi-1.0, \
- io.openliberty.jakarta.cdi-3.0, \
- io.openliberty.jakarta.interceptor-2.0, \
- com.ibm.websphere.appserver.jdbc-4.2; ibm.tolerates:="4.3"
+ com.ibm.websphere.appserver.webProfile-9.0,\
+ com.ibm.websphere.appserver.transaction-2.0,\
+ io.openliberty.ejbHome-4.0
 kind=beta
-edition=core
+edition=base
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature:14]: How can this be in base when all of it's contents are in core?

**Pull request merged**